### PR TITLE
[docs] Removed PostGIS from docs

### LINKED ISSUE #21413: [Docs] Remove PostGIS documentation from DOCS
### Description

We list PostGIS on following pages: 

- https://docs.yugabyte.com/preview/explore/ysql-language-features/pg-extensions/#other-extensions 
- https://docs.yugabyte.com/preview/explore/ysql-language-features/pg-extensions/extension-postgis/ 

This is an unsupported extension, should be removed from the docs pages.

### Warning: Please confirm that this issue does not contain any sensitive information

- [X] I confirm this issue does not contain any sensitive information.

### PR DESCRIPTION
**Pull request by aishwarya24**
resolves #21413 
Removed PostGIS and pgsql-postal

@netlify /preview/explore/ysql-language-features/pg-extensions/

### CODE REVIEW

**Comment by ddhodge** [comment]
There is a mention in Voyager docs.

**Comment by aishwarya24** [comment]
> There is a mention in Voyager docs.

Yes, but that links to https://postgis.net/ and not our YB extension. So I didn't remove it.

File: docs/content/preview/explore/ysql-language-features/pg-extensions/extension-pgsql-postal.md [hunk 1]
```diff
@@ -21,7 +21,35 @@ First install `libpostal` [from source](https://github.com/openvenues/libpostal)
 `make -j$(nproc) && sudo make install`
 ```
 
-To build `pgsql-postal` against the correct PostgreSQL version for YugabyteDB compatibility, install PostgreSQL 11 on your system as described in the [PostGIS example](../extension-postgis/).
+To build `pgsql-postal` against the correct PostgreSQL version for YugabyteDB compatibility, install PostgreSQL 11 on the your system using the following instructions applicable to your operating system.
+
```
Comment (ddhodge) [inline docs/content/preview/explore/ysql-language-features/pg-extensions/extension-pgsql-postal.md:25]: @sushantrmishra CentOS is deprecated. Do we need to provide instructions at all? ie just say "..., install PostgreSQL 11 on your system."

File: docs/content/preview/explore/ysql-language-features/pg-extensions/extension-pgsql-postal.md [hunk 2]
```diff
@@ -21,7 +21,35 @@ First install `libpostal` [from source](https://github.com/openvenues/libpostal)
 `make -j$(nproc) && sudo make install`
 ```
 
-To build `pgsql-postal` against the correct PostgreSQL version for YugabyteDB compatibility, install PostgreSQL 11 on your system as described in the [PostGIS example](../extension-postgis/).
+To build `pgsql-postal` against the correct PostgreSQL version for YugabyteDB compatibility, install PostgreSQL 11 on the your system using the following instructions applicable to your operating system.
```
Comment (ddhodge) [inline docs/content/preview/explore/ysql-language-features/pg-extensions/extension-pgsql-postal.md:24]:
```suggestion
To build `pgsql-postal` against the correct PostgreSQL version for YugabyteDB compatibility, install PostgreSQL 11 on your system using the following instructions applicable to your operating system.
```

**Review by ddhodge — APPROVED** [review]

**Pull request merged**